Hunting Queries/SecurityEvent/ADAccountLockouts.yaml

```diff
@@ -0,0 +1,21 @@
+id: e7642e6e-cf27-46ec-a4b9-e4475228fead
+name: AD Account Lockout
+description: |
+  'Detects Active Directory account lockouts'
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+tactics:
+  - InitialAccess
+  - CredentialAccess
+relevantTechniques:
+  - T1531
+query: |
+let timeframe = 7d;
+SecurityEvent
+| where TimeGenerated >= ago(timeframe)
+| where EventID == 4740
+| extend AccountCustomEntity = tostring(split(Account, '\\', 1)), SourceDomainController = Computer, HostCustomEntity = TargetDomainName
+| summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), LockoutsCount = count() by AccountCustomEntity, HostCustomEntity, SourceDomainController
+| extend timestamp = StartTime
```

Comment:
In general, good stuff.  We usually bring thru the common name and add in the extended name so we don't lose the specific context of the original event.  Here is what we can do and I put some comments in to help understand why we bring a value thru, you can choose to leave those in or take them out.

let timeframe = 4h;
SecurityEvent
| where TimeGenerated >= ago(timeframe)
| where EventID == 4740
// Bring thru 
// Activity <-- this is the event id with the friendly name and is helpful for context
// TargetSid <-- strong identifer for the user account
// TargetDomainName <-- Host where the request came from
// SourceComputerId <-- this is a strong identifer to use with other tables such as Heartbeat to get something like the IP at the time
// Computer as SourceDomainController <-- this is the DC where the Event was logged that is associated to the account that locked out
| summarize StartTime = min(TimeGenerated), EndTime = max(TimeGenerated), LockoutsCount = count() by Activity, Account, TargetSid, TargetDomainName, SourceComputerId, SourceDomainController = Computer
| extend timestamp = StartTime, AccountCustomEntity = Account, HostCustomEntity = TargetDomainName